_posts/2018-11-16-webhooks-companies.markdown

```diff
@@ -0,0 +1,135 @@
+---
+permalink: v1/webhooks/companies
+title: Companies Webhooks
+layout: sidebar
+---
+
+# Companies Webhooks
+
+## Get Company Webhook Notifications
+
+[Webhooks Overview](/v1/webhooks/about)
+
+**Endpoint**: `GET /v1/companies/:company_id`
```

Comment:
As discussed, kill this section in favor of linking the "Entity" to it's corresponding "Resource"

Reply:
Making the entity name itself the link gives no indication that it is a link, even on hover. I opted for having a link in parenthesis next to it:

<img width="259" alt="screenshot 2018-11-28 14 29 26" src="https://user-images.githubusercontent.com/2102795/49183391-085fde80-f31a-11e8-85fd-aea432effa20.png">